packages/controller-utils/src/constants.ts

```diff
@@ -3,6 +3,7 @@ import { NetworkType, NetworksTicker, ChainId, NetworkId } from './types';
 export const RPC = 'rpc';
 export const FALL_BACK_VS_CURRENCY = 'ETH';
 export const IPFS_DEFAULT_GATEWAY_URL = 'https://cloudflare-ipfs.com/ipfs/';
+export const TOKEN_URI_IPFS = 'ipfs://*';
```

Comment:
Nit: It seems like this is only used for tests. If that's the case, does it make sense to only keep it in `NftController.test.ts`? That way we're not adding things to the public API that aren't used.

Reply:
You are right! Will update it!